File: task03view/app/src/main/java/by/mankevich/task03view/servicejava/IOUtils.java
package by.mankevich.task03view.servicejava;

import java.io.IOException;
import java.io.InputStream;
import java.io.Reader;

public class IOUtils {
    public static void closeQuietly(InputStream in){
        try{
            in.close();
        } catch (IOException e) {

        }
    }

    public static void closeQuietly(Reader reader){
        try{
            reader.close();
        } catch (IOException e) {

        }
    }
}

File: task03view/app/src/main/java/by/mankevich/task03view/servicekotlin/IOUtils.kt
package by.mankevich.task03view.servicekotlin

import java.io.IOException
import java.io.InputStream
import java.io.Reader

class IOUtils {
    fun closeQuietly(inputStream: InputStream) {
        try {
            inputStream.close()
        } catch (e: IOException) {
        }
    }

    fun closeQuietly(reader: Reader) {
        try {
            reader.close()
        } catch (e: IOException) {
        }
    }
}